packages/celotool/src/lib/prometheus.ts

```diff
@@ -254,10 +273,6 @@ async function createPrometheusGcloudServiceAccount(
     await execCmdWithExitOnFailure(
       `gcloud projects add-iam-policy-binding ${gcloudProjectName} --role roles/monitoring.metricWriter --member serviceAccount:${serviceAccountEmail}`
     )
-    // Prometheus needs roles/compute.viewer to discover the VMs asking GCE API
-    await execCmdWithExitOnFailure(
```

Comment:
Why can this be removed?
Would it make sense to manually change the current permissions to revoke if it's no longer needed?

Reply:
In the past we used prometheus to scrape the metrics from the workloads running as VMs.
There are not workloads running as VMs with metrics, so this permission is not needed any longer.